src/js/bootstrap-slider.js

```diff
@@ -1148,6 +1148,9 @@ const windowIsDefined = (typeof window === "object");
 
 				this.handle1.style[this.stylePos] = positionPercentages[0]+'%';
 				this.handle1.setAttribute('aria-valuenow', this._state.value[0]);
+				if (isNaN(this.options.formatter(this._state.value[0])) ) {
+					this.handle1.setAttribute('aria-valuetext', this.options.formatter(this._state.value[0]));
```

Comment:
> about handle2... 
> What is it's purpose? 

Purpose is the use case where we have a 2 handle slider (see Example 2 in our examples page - http://seiyria.com/bootstrap-slider/)